File: src/com/cs/solaire/Coupure.cs
using System;
namespace src.com.cs {
public class Coupure {
 {
    private object datecoupure;
    private object idsource;
    private object heurecoupure;

    public Coupure(object datecoupure, object idsource, object heurecoupure) {
        this.datecoupure = datecoupure;
        this.idsource = idsource;
        this.heurecoupure = heurecoupure;
    }
    public object Datecoupure {
        get { return datecoupure; }
        set { datecoupure = value; }
    }
    public object Idsource {
        get { return idsource; }
        set { idsource = value; }
    }
    public object Heurecoupure {
        get { return heurecoupure; }
        set { heurecoupure = value; }
    }
}
    }
}

File: src/com/classe/solaire/Coupure.java
package com.classe.solaire;

import java.lang.Object;

public class Coupure {
  private Object datecoupure;

  private Object idsource;

  private Object heurecoupure;

  public Coupure(Object datecoupure, Object idsource, Object heurecoupure) {
    this.datecoupure = datecoupure;
    this.idsource = idsource;
    this.heurecoupure = heurecoupure;
  }

  public Object getDatecoupure() {
    return datecoupure;
  }

  public void setDatecoupure(Object datecoupure) {
    this.datecoupure = datecoupure;
  }

  public Object getIdsource() {
    return idsource;
  }

  public void setIdsource(Object idsource) {
    this.idsource = idsource;
  }

  public Object getHeurecoupure() {
    return heurecoupure;
  }

  public void setHeurecoupure(Object heurecoupure) {
    this.heurecoupure = heurecoupure;
  }
}
